BUILDING-RELEASES.adoc

```diff
@@ -0,0 +1,54 @@
+== How To Build Releases Using Maven Release Plugin and Nexus Repositories
+
+The following steps will tell you what you need to do in order to build a release for public consumption.
+
+Before you can do anything, you must ensure your Maven environment and project are configured properly.
+
+* Put your Maven public repository credentials in your ~/.m2/settings.xml file. For example, the Hawkular parent pom defines the public maven repository that we use for releases under the name "jboss-releases-repository", so in your settings.xml you will need something like this:
+[source,xml]
+<servers>
+  <server>
+    <id>jboss-releases-repository</id>
+    <username>your.maven.repository.username</username>
+    <password>your.maven.repository.password</password>
+  </server>
+</servers>
+
+* Make sure your Maven project's main pom.xml defines an <scm> section with at least a <developerConnection> entry. Here's an example:
```

Comment:
This is not enough, IMHO. You have to have the <scm> in EVERY pom in every module sharing a single repo otherwise release plugin will mess up your git URLs by trying to create subpaths of it corresponding to the individual sub modules.